Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Culling.h

```diff
@@ -88,6 +88,11 @@ namespace AZ
                 //! Suggest setting to: 0.5f*localAabb.GetExtents().GetMaxElement()
                 float m_lodSelectionRadius = 1.0f;
 
+                // the minimum possibe area a sphere enclosing a mesh projected onto the screen should have before it is culled.
```

Comment:
Should add the previous comment explaining why these defaults were chosen for 1080p

Reply:
Done.